main.py

```diff
@@ -1,10 +1,59 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for user, info in data.items():
+        nickname = user
+        email = info.get("email")
+        bio = info.get("bio")
+
+        if info["guild"]:
+            guild_name = (info["guild"]["name"]
+                          if info["guild"]["name"] else None)
+            guild_description = (info["guild"]["description"]
+                                 if info["guild"]["description"] else None)
```

Comment:
You can use dict.get() method instead of ternary and try to remove redundant variables